### PR TITLE
add Geoapify Provider

### PR DESCRIPTION
**Pull request by CarolineVantim**
Added provider for communication with the Geoapify API.

### CODE REVIEW

**Review by smeijer — CHANGES_REQUESTED** [review]
Hi! This looks great. Can you revert the changes to package.json so we can merge?

**Comment by CarolineVantim** [comment]
Changes made

**Comment by CarolineVantim** [comment]
Stephan, could you check if my adjustments are correct?

File: src/providers/__tests__/geoapifyProvider.spec.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      params: {
+        apiKey: process.env.GEOAPIFY_API_KEY,
+        'accept-language': 'nl',
+      },
+    });
+
+    const results = await provider.search({ query: 'Chicago' });
+    t.is(
+      results[0].label,
+      '1214-1224 West Van Buren Street, Chicago, IL 60607, United States of America'
```
Comment (smeijer) [inline src/providers/__tests__/geoapifyProvider.spec.js:36]:
```suggestion
      '1214-1224 West Van Buren Street, Chicago, IL 60607, United States of America',
```

**Review by smeijer — APPROVED** [review]
Sorry, completely slipped my mind. Things are looking good now. Thanks so much!

**Pull request merged**